### PR TITLE
Updated web/ folder with API config from overrides.json

### PR DESCRIPTION
**Pull request by rshbhgrg**
Refs https://github.com/TryGhost/Ghost/issues/9866

- exports `getApiPath` method from service/url/utils
- uses `getApiPath` in web/parent-app to configure API paths for supported versions

### CODE REVIEW

File: core/server/web/parent-app.js [hunk 1]
```diff
@@ -38,9 +39,9 @@ module.exports = function setupParentApp(options = {}) {
     // API
     // @TODO: finish refactoring the API app
     // @TODO: decide what to do with these paths - config defaults? config overrides?
-    parentApp.use('/ghost/api/v0.1/', require('./api/v0.1/app')());
-    parentApp.use('/ghost/api/v2/content/', require('./api/v2/content/app')());
-    parentApp.use('/ghost/api/v2/admin/', require('./api/v2/admin/app')());
+    parentApp.use(urlUtils.getApiPath('deprecated'), require('./api/v0.1/app')());
+    parentApp.use(urlUtils.getApiPath('active'), require('./api/v2/content/app')());
+    parentApp.use(urlUtils.getApiPath('active', true), require('./api/v2/admin/app')());
```
Comment (kirrg001) [inline core/server/web/parent-app.js]: > urlUtils.getApiPath('active', true)

It's always hard to guess what "true" here means.
Let's use the option pattern?

`getApiPath({WHAT: active, admin: true})`

Comment (rishabhgrg) [reply core/server/web/parent-app.js]: 👍 Yeah makes sense. I think might be better to have a separate PR which takes care of that as that will changes across the board along with tests, wdyt @kirrg001 ?

Comment (kirrg001) [reply core/server/web/parent-app.js]: Agree

Comment (rishabhgrg) [reply core/server/web/parent-app.js]: Actually the tests are not affected, only the method `urlFor` gets affected. 🤔

**Pull request merged**